train/create_augmentation_targets.py

```
@@ -0,0 +1,212 @@
+# Helpers
+# ---------------------------------------------------------------------------
+
+def load_dataset(path: str) -> pd.DataFrame:
+    df = pd.read_csv(path)
+    df = df.dropna(subset=["sentence"])
+    df["sentence"] = df["sentence"].astype(str).str.strip()
+    df = df[df["sentence"] != ""]
+    return df
+
+
+def pick_seed_examples(sentences: list[str], n: int, rng: random.Random) -> list[str]:
```

Ruff (0.15.2)

[warning] Unused function argument: `rng`

(ARG001)